common/scala/src/main/scala/whisk/common/Logging.scala

```diff
@@ -212,6 +214,15 @@ object MetricEmitter {
 
   val metrics = Kamon.metrics
```

Comment:
Not sure if it would be safe to add to `MetricEmitter` as first reference to `MetricEmitter` object itself would trigger loading of `metrics`  and we would probably need to have config provided and Kamon started before that happens. 

So would be safer to have a new object hosting this method